index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container">
+        <div class="logoholder">
+            <b class="mylogo">My Logo</b>
+        </div>
+        <div class="menuholder">
+            <img src="menuicon.png" alt="test">
+
+        </div>
```

Comment:
- [ ] It would be better to fix the nav, the hamburger is out of position currently
![Screenshot from 2021-08-12 21-38-12](https://user-images.githubusercontent.com/46388113/129273678-912a986d-6c7f-4be5-8d1b-e3a7e1029423.png)